resources/app.yaml

```diff
@@ -19,7 +19,7 @@ hooks:
             restartPolicy: OnFailure
             containers:
               - name: hook
-                image: quay.io/gravitational/debian-tall:stretch
+                image: quay.io/gravitational/debian-grande:stretch
```

Comment:
Do we have to use debian-grande here? This will increase the tarball even further, right?

Reply:
I believe we already have grande in telegraf, but I will take a look at how to use only tall in both.

Reply:
@UnderGreen If we already have it, that's fine. If it's easy to use tall instead, that's ideal, but if it's too much work, I wouldn't bother. Esp. for 5.5 stack.